common/Resources/ti.internal/extensions/node/process.js

```diff
@@ -253,3 +255,103 @@ Ti.App.addEventListener('uncaughtException', function (event) {
 });
 
 export default process;
+
+// Use a nice predictable class/structure for our timers
+// JS engine should be able to optimize easier
+class Timer {
+	constructor(fun, args) {
```

Comment:
`func` is a more clear abbreviation here

```suggestion
	constructor(func, args) {
```